components/utils/src/templates.rs

```diff
@@ -19,8 +19,23 @@ pub fn render_template(name: &str, tera: &Tera, context: &Context, theme: Option
             .map_err(|e| e.into());
     }
 
-    if name == "index.html" || name == "section.html" || name == "page.html" {
-        return Ok(String::new());
+    if name == "index.html" {
+        return tera
+            .render("index.html", &{})
+            .map_err(|e| e.into());
+    }
+
+
+    if name == "section.html" {
+        return tera
+            .render("section.html", &{})
+            .map_err(|e| e.into());
+    }
+
+    if name == "page.html" {
+        return tera
+            .render("page.html", &{})
+            .map_err(|e| e.into());
```

Comment:
Hmm I remember now why I didn't make them a built-in: to allow themes. That's a bit trickier to handle, I think I'll do it after merging